corehq/apps/sso/models.py

```diff
@@ -136,6 +137,19 @@ def get_sso_exempt_users(self):
             email_domain__identity_provider=self,
         ).values_list('username', flat=True)
 
+    def get_login_url(self, username=None):
+        """
+        Gets the login endpoint for the IdentityProvider based on the protocol
+        being used. Since we only support SAML2 right now, this redirects to
+        the SAML2 login endpoint.
```

Comment:
Optional: did you consider just making this explicitly SAML2 until there's a need for multiple versions? this comment has a little bit of "yagni" feel to it (though I have no idea what our roadmap looks like for adding other options)

Reply:
it's still a handy shortcut for the username so I don't think this is full "yagni", just a thoughtful comment. Also, we will be supporting other Identity Providers in the future, and it's unclear what their support will be

Reply:
Got it. Also probably I could have been more clear in my comment: I just meant renaming the function to `get_saml2_login_url`.